src/Tribe/Attendees.php

```diff
@@ -151,16 +151,25 @@ public function print_checkedin_totals() {
 	 * Returns the full URL to the attendees report page.
 	 *
 	 * @since 4.6.2
+	 * @since TBD - Added $post_id parameter to allow for conversion of provisional ID's into regular Post ID's.
 	 *
 	 * @param WP_Post $post
+	 * @param int $post_id The Post ID of the event.
 	 *
 	 * @return string
 	 */
 	public function get_report_link( $post ) {
+		// Convert provisional ID's into regular Post ID's.
+		if ( class_exists( '\TEC\Events\Custom_Tables\V1\Models\Occurrence', false ) ) {
```

Comment:
@EugeneKyale do you think this case could occur in other places? I see at least [one other place](https://github.com/the-events-calendar/event-tickets/blob/8661b22a06dbaff45d61b61e4e2e710dd17bc787/src/Tribe/Tickets.php#L424).

Would it make sense to make a wrapper function for the id normalization we could call in different places?

Reply:
@juanfra yeah it makes sense to have a helper function so can ensure consistency in handling IDs throughout our code. Which class do you think is best suited to host the helper function?